app/facades/fdc_facade.rb

```diff
@@ -0,0 +1,10 @@
+class FdcFacade
+  def self.get_search_results(food)
+    json = FdcAPIService.food_search(food)
+    total_results = json[:totalHits]
+
+    food_objects = json[:foods].map do |food|
```

Comment:
Great job only hitting the external api service once. One way you could have broken up the food objects from the total_results is to have built a hash at the end of this method that returned `{results: total_results, foods: food_objects}` and then you could have extracted those values into instance variables to pass to your view.